content/get-haiku/r1beta2/_index.md

```diff
@@ -0,0 +1,111 @@
+
+(If you are looking for the nightly images, they can be found at <a href="https://download.haiku-os.org">download.haiku-os.org</a>.)
+
+## Upgrading from R1/beta1
+
+There is an upgrade path available for users that are currently using Haiku R1/beta1. If you want to do this upgrade, you will have to use the Terminal (not SoftwareUpdater!) to issue some commmands.
+
+*Warning: only upgrading from Beta 1 is supported. It may be possible to upgrade from prior development images, but this is untested and you may run into unknown problems.*
+
+The following commands will replace your existing "Haiku" and "HaikuPorts" repositories with their R1/beta2 equivalents:
+
+```shell script
+pkgman add-repo https://eu.hpkg.haiku-os.org/haiku/r1beta2/$(getarch)/current
+pkgman add-repo https://eu.hpkg.haiku-os.org/haikuports/r1beta2/$(getarch)/current
+```
+
+If this is successful, you may then run a `full-sync` to perform the actual upgrade:
+
+```shell script
+pkgman full-sync
+```
+
+After that step has completed succesfully, you should immediately reboot the system. You can do this from the Deskbar, or by running ```shutdown -r``` in Terminal.
+
+### Download a Haiku Image
+
+The provided images are in our `anyboot` format. They can be written directly to a <a href="/guides/installing/making_haiku_usb_stick">USB flash drive</a>, an empty disk, or <a href="/get-haiku/burn-cd">written to DVD</a> media.<br />
+The images can be used "live" or used to install Haiku to another disk/partition of your choice.<br />
+They can also be used directly from QEMU or installed in other <a href="/guides/virtualizing">virtual machine</a>.
+
+### Direct Download Locations
+
+If you choose to download an image, please select from the list of available mirrors below. Mirrors are in no specific order.
+<p><strong>NOTE: Not all mirrors may be online. Please choose another mirror if you encounter a failure.</strong></p>
+
+<div class="nolinks">
+<table id="mirrors" class="table thead-dark table-hover">
+<thead style="font-weight: bold;">
+<tr>
+<td>Mirror</td>
+<td title="BeOS API + binary compatible">32-bit</td>
+<td title="BeOS API compatible">64-bit</td>
+</tr>
+</thead>
+<tbody>
+    <tr>
+      <td class="location"><b>Location:</b> East Coast, United States <br/>Provided by: <a target="_blank" class="ext" href="https://haiku-inc.org" title="Haiku, Inc.">Haiku, Inc.</a></td>
+      <td><a class="track" href="https://s3.wasabisys.com/haiku-release/r1beta2/haiku-r1beta2-x86_gcc2_hybrid-anyboot.zip">zip</a></td>
+      <td><a class="track" href="https://s3.wasabisys.com/haiku-release/r1beta2/haiku-r1beta2-x86_64-anyboot.zip">zip</a></td>
+    </tr>
+    <tr>
+      <td class="location"><b>Location:</b> Amsterdam, Netherlands <br/>Provided by: <a target="_blank" class="ext" href="https://haiku-inc.org" title="Haiku, Inc.">Haiku, Inc.</a></td>
+      <td><a class="track" href="https://cdn.haiku-os.org/haiku-release/r1beta2/haiku-r1beta2-x86_gcc2_hybrid-anyboot.zip">zip</a></td>
```

Comment:
Amsterdam links seem broken.

Reply:
When I uploaded this, both links were broken. CC @nielx, did you figure out how to upload to here?